backend/src/StdLibExecution/Libs/Json.fs

```diff
@@ -202,11 +203,20 @@ let rec serialize
               "Incorrect record type"
               [ "actual", actualTypeName; "expected", typeName ]
           | _ ->
+            debuG
+              "This seems to be breaking local-exec load-packages-dark"
+              (dval, typ, LibExecution.DvalReprDeveloper.dvalTypeName dval)
```

Comment:
TODO: remove this `debuG` before merging, ideally - I haven't tested yet to see if this is fixed